dev/ci/ci-color.sh

```diff
@@ -5,6 +5,17 @@ source ${ci_dir}/ci-common.sh
 
 Color_CI_DIR=${CI_BUILD_DIR}/color
 
+# Setup Bignums
+
+source ${ci_dir}/ci-bignums.sh
```

Comment:
> I don't think we want to use source to install packages

Why not ? I personally find it cleaner this way, with stuff about bignums located in `ci-bignums.sh` rather that who knows elsewhere. But I won't fight over this. I'll implement what you ask if I've 5min today, otherwise feel free to do what you what if that gets merged.